lib/valkyrie/indexers/access_controls_indexer.rb

```diff
@@ -8,12 +8,35 @@ def initialize(resource:)
 
     def to_solr
       return {} unless resource.respond_to?(:read_users)
+      conf = config
```

Comment:
Why not just call config directly?

Reply:
And maybe we should move this to Valkyrie's config?

Reply:
I'm doing this to avoid 4 instantiations of the hash.  It's a minor optimization.

Reply:
If it's significant (I doubt it), then caching the config seems better.

Reply:
Okay, I just added it as an argument to the initializer.